tools/site_admin/rename_pages.php

```diff
@@ -341,14 +341,22 @@
 
                 // database
                 echo "    database:";
-                $query = "
+
+                // first the project table
+                $project_query = sprintf(
+                    "
                     UPDATE $projectid
-                    SET fileid='$new_fileid', image='$new_image'
-                    WHERE fileid='$old_fileid' AND image='$old_image'
-                ";
-                echo $query;
+                    SET fileid = '%s', image = '%s'
+                    WHERE fileid = '%s' AND image='%s'
```

Comment:
Nit: Inconsistent spacing around `image='%s'` (you've used spaces before/after `=` previously).